torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -1585,7 +1579,13 @@ void TensorExprKernel::compile() {
       stmt_,
       bufferArgs_,
       device_,
-      kernel_func_name_);
+      kernel_func_name_,
+      pre_alloc_);
+
+  if (pre_alloc_) {
```

Comment:
I understand. This is what we need to go through to build good things. Thanks for letting me know. Otherwise it would take more time and effort for me to maintain this PR.